creator-node/scripts/start.sh

```diff
@@ -7,13 +7,18 @@ if [[ "$WAIT_HOSTS" != "" ]]; then
     /usr/bin/wait
 fi
 
-if [[ -z "$logglyDisable" ]]; then
-    if [[ -n "$logglyToken" ]]; then
-        logglyTags=$(echo $logglyTags | python3 -c "print(' '.join(f'tag=\\\\\"{i}\\\\\"' for i in input().split(',')))")
-        mkdir -p /var/spool/rsyslog
-        mkdir -p /etc/rsyslog.d
-        sed -i '1s|^|$MaxMessageSize 64k\n|' /etc/rsyslog.conf
-        cat >/etc/rsyslog.d/22-loggly.conf <<EOF
+# enable rsyslog if not set to 0
+: "${enableRsyslog:=1}"
```

Comment:
> just curious - why 0/1 vs true/false?

Since I didn't want to account for lowercase/uppercase differences, truncated values of "true", etc.

This is simple logic of 0 == off, 1 == on. Thoughts?

> also suggestion on the phrasing of the comment "enable rsyslog if not explicitly disabled by another logging transport"

It's not entirely accurate since the Logspout doesn't disable rsyslog. Instead this PR disables it:

https://github.com/AudiusProject/audius-docker-compose/pull/17

Thoughts on me adding: "enable rsyslog if not explicitly disabled by audius-docker-compose"?

Reply:
I strongly think we should use true / false like any other env vars, this introduces inconsistent variable conventions in the code.

Reply:
Indeed, we're purposefully breaking convention for these reasons:

* https://github.com/AudiusProject/audius-protocol/pull/3124#discussion_r882212634

Going forward a new convention will be much easier to read and manage.

Reply:
Ok, after a bit of experimentation. What are your thoughts on this syntax:

```
# enableRsyslog=true
# enableRsyslog=false

: "${enableRsyslog:=true}"

if $enableRsyslog; then
    echo 1;
fi
```

This only works if we restrict ourselves to solely using `true` or `false`.

How does this approach look to both of you? @dmanjunath @cheran-senthil

Reply:
yeah this looks good to me